docs/concepts/apiserver_admission_plugins.md

```diff
@@ -4,7 +4,7 @@ title: APIServer Admission Plugins
 
 # Admission Plugins
 
-Similar to the kube-apiserver, the gardener-apiserver comes with a few in-tree managed admission plugins.
+Similarly to the kube-apiserver, the gardener-apiserver comes with a few in-tree managed admission plugins.
```

Comment:
Isn't this wrong? We are not describing a verb here, hence we shouldn't use adverbs. Am I missing something?

Reply:
> Isn't this wrong? We are not describing a verb here, hence we shouldn't use adverbs. Am I missing something?

In regards to the usage of "Similar to" vs "Similarly to", I have found some conflicting materials. I personally feel that using "Similarly to the kube-apiserver..." conveys "In a way similar to the kube-apiserver...", while "Similar to the kube-apiserver..." directly compares the two "The gardener-apiserver is similar to the kube-apiserver, which is why it comes with a few...". In that sense, I think that we should use "Similarly to...".

We could also avoid this as a whole by changing the sentence structure.

Reply:
I would not start to over-engineer here. We could also just keep it as is :)

Reply:
Alright, that works for me. Do you have any other suggestions regarding the contents of the PR?

Reply:
Thanks, I haven't checked most of the remaining changes yet. Perhaps you could adapt this one (which probably results in a smaller diff), and then I/others can check again :)

Reply:
I have went over all other uses of "similar to" and "similarly to" and made the necessary changes.

Reply:
@n-boshnakov I still see a lot of "similarly to" changes.

Reply:
> @n-boshnakov I still see a lot of "similarly to" changes.

I feel that those are necessary to include. If you want me to remove them completely, I can do so.

Reply:
Alright, will do.